backend/DatabaseScripts/SqlScripts/0041_Recreate_materialized_view_graphql_pool_mean_apys_due_to_double_cast.sql

```diff
@@ -0,0 +1,73 @@
+/*
+Recreates graphql_pool_mean_apys from migration script 0031_CreateMaterializedView_graphql_pool_mean_apys.sql.
+APY's values from view was mapped to postgres type 'numeric' which couldn't be casted to C# 'double' or 'decimal' 
+used by Entity Framework when mapping.
+Changes below cast calculated values to PostgreSQL 'double' values such that they can be mapped.
+
+See link for PostgreSQL numeric datatypes
+https://www.postgresql.org/docs/current/datatype-numeric.html
+*/
+
+DROP MATERIALIZED VIEW IF EXISTS graphql_pool_mean_apys;
+
+create materialized view graphql_pool_mean_apys as
```

Comment:
I'm not sure how well this view will scale. It might not be important either.

The reason is we have a lot of sub-queries, so if data_30_days grows in size, the sub-queries will become more expensive. It's hard to determine without knowing the size of datasets we'll be working with.

Is the view used in any of the automated tests? If not, it might be a good idea to add a test, to ensure it will if any of the dependent tables/columns are modified.

Reply:
Talk with @ali-concordium and will merge without integration test. Test has been done manually against environment.